wave_lang/kernel/wave/codegen/emitter.py

```diff
@@ -54,8 +54,8 @@
 
 
 def _get_upper_bound(expr: Any) -> Optional[Attribute]:
-    res = subs_idxc(sympy.sympify(expr))
```

Comment:
Does removing the `sympify` call possibly change some simplifications performend on expr or is this a non functional change?

i.e. could this impact the code we generate?

Reply:
`sYmpify` just converts basic types to sympy types, it should have no effect.